File: FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/ApiName.java
/*
 * © 2016-2019 FlowCrypt Limited. Limitations apply. Contact human@flowcrypt.com
 * Contributors: DenBond7
 */

package com.flowcrypt.email.api.retrofit;

/**
 * List of all API which uses in the RETROFIT.
 *
 * @author Denis Bondarenko
 * Date: 10.03.2015
 * Time: 13:51
 * E-mail: DenBond7@gmail.com
 */
public enum ApiName {
  /*attester.flowcrypt.com*/
  POST_TEST_WELCOME,
  POST_INITIAL_REQUEST,
  POST_INITIAL_CONFIRM,
  POST_INITIAL_LEGACY_SUBMIT,
  POST_LOOKUP_EMAIL_SINGLE,
  POST_LOOKUP_EMAIL_MULTIPLY,
  POST_REPLACE_REQUEST,
  POST_REPLACE_CONFIRM,
  GET_LOOKUP,

  /*api.cryptup.io*/
  POST_HELP_FEEDBACK,
  POST_LINK_MESSAGE,
  POST_MESSAGE_REPLY
}

File: FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/ApiName.kt
/*
 * © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
 * Contributors: DenBond7
 */

package com.flowcrypt.email.api.retrofit

/**
 * List of all API which uses in the RETROFIT.
 *
 * @author Denis Bondarenko
 * Date: 10.03.2015
 * Time: 13:51
 * E-mail: DenBond7@gmail.com
 */
//todo-denbond7 need to remove this class after refactoring
enum class ApiName {
  /*flowcrypt.com/attester*/
  GET_PUB,

  /*flowcrypt.com/api*/
  POST_HELP_FEEDBACK,
  POST_LINK_MESSAGE,
  POST_MESSAGE_REPLY,
  POST_LOGIN,
  POST_GET_DOMAIN_RULES
}
